Lab5.ipynb

```diff
@@ -0,0 +1,527 @@
+       "ee3         0.006981  0.001447 -0.003464  0.793991  0.844699  1.000000   \n",
+       "d2         -0.003817 -0.002371  0.001273 -0.538750 -0.567445 -0.306321   \n",
+       "angularity  0.032309  0.000908 -0.008891  0.529638  0.427854  0.433308   \n",
+       "t1         -0.010783 -0.002803  0.002601  0.010450  0.126657  0.138780   \n",
+       "t2         -0.027798 -0.001568  0.000465 -0.533355 -0.517926 -0.250764   \n",
+       "t3         -0.033102 -0.001840  0.001326 -0.621826 -0.620038 -0.350501   \n",
+       "t21        -0.027596 -0.000351 -0.000321 -0.585519 -0.613574 -0.323984   \n",
+       "t32        -0.024690 -0.000405  0.002848 -0.459640 -0.495083 -0.412092   \n",
+       "KtDeltaR    0.024390 -0.002086 -0.003087  0.708356  0.614471  0.459877   \n",
+       "\n",
+       "                  d2  angularity        t1        t2        t3       t21  \\\n",
+       "pt         -0.003817    0.032309 -0.010783 -0.027798 -0.033102 -0.027596   \n",
+       "eta        -0.002371    0.000908 -0.002803 -0.001568 -0.001840 -0.000351   \n",
+       "phi         0.001273   -0.008891  0.002601  0.000465  0.001326 -0.000321   \n",
+       "mass       -0.538750    0.529638  0.010450 -0.533355 -0.621826 -0.585519   \n",
+       "ee2        -0.567445    0.427854  0.126657 -0.517926 -0.620038 -0.613574   \n",
+       "ee3        -0.306321    0.433308  0.138780 -0.250764 -0.350501 -0.323984   \n",
+       "d2          1.000000   -0.184381 -0.022127  0.554533  0.622112  0.619491   \n",
+       "angularity -0.184381    1.000000 -0.052497 -0.152266 -0.196199 -0.155062   \n",
+       "t1         -0.022127   -0.052497  1.000000  0.326725  0.238284 -0.002957   \n",
+       "t2          0.554533   -0.152266  0.326725  1.000000  0.936640  0.926364   \n",
+       "t3          0.622112   -0.196199  0.238284  0.936640  1.000000  0.897178   \n",
+       "t21         0.619491   -0.155062 -0.002957  0.926364  0.897178  1.000000   \n",
+       "t32         0.325022   -0.190053 -0.146209  0.129887  0.443389  0.194428   \n",
+       "KtDeltaR   -0.392433    0.406357 -0.271773 -0.635300 -0.639734 -0.572633   \n",
+       "\n",
+       "                 t32  KtDeltaR  \n",
+       "pt         -0.024690  0.024390  \n",
+       "eta        -0.000405 -0.002086  \n",
+       "phi         0.002848 -0.003087  \n",
+       "mass       -0.459640  0.708356  \n",
+       "ee2        -0.495083  0.614471  \n",
+       "ee3        -0.412092  0.459877  \n",
+       "d2          0.325022 -0.392433  \n",
+       "angularity -0.190053  0.406357  \n",
+       "t1         -0.146209 -0.271773  \n",
+       "t2          0.129887 -0.635300  \n",
+       "t3          0.443389 -0.639734  \n",
+       "t21         0.194428 -0.572633  \n",
+       "t32         1.000000 -0.222624  \n",
+       "KtDeltaR   -0.222624  1.000000  "
+      ]
+     },
+     "execution_count": 74,
+     "metadata": {},
+     "output_type": "execute_result"
+    }
+   ],
+   "source": [
+    "qcd_file.corr()"
```

Comment:
Bottom table:
- I think this is a Pandas correlation table/matrix, right? Visually looks good, printed nicely.
- What specifically were you trying to describe/show using this table? I would include a description after to help explain. Is this to show relationships between the different variables within the background dataset or something else?

I know you weren't done at time of uploading draft, but I think the initial descriptions in the beginning were really good so far, and the setup of the plots are good so far as well.
Now, some additional descriptions (like for the table and possibly more about the graphs) would be good; if possible, more information on how Higgs data is collected (or simulated) (looking at the lab 5 instructions).
I am also wondering if there are other plots besides histograms that we can use to look at the most important parameters. Any ideas? I am still looking around with this tonight as well.